Ensure sending dir path and not file path (#2228)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue causing any command to crash when unable to detect local repository properties.
* Fixed an issue where running in a private gitlab repo caused a warning message to be shown multiple times.
* Added a new validation to the **validate** command to verify that markdown and python files do not contain words related to copyright section.

> * Fixed an issue causing lint command to crash when sending a file path instead of a directory path.

Reviewer: Did it also crash for `.yml` files?

> * Fixed an issue causing lint command to crash when sending a file path instead of a directory path.

Reviewer 2: ```suggestion
* Fixed an issue where **lint** crashd when provided an input file path (expecting a directory).
```
